apps/web/src/utils/auth.ts

```diff
@@ -109,26 +109,37 @@ const providers: Provider[] = [
 
 // If we have an OAuth provider configured, we add it to the list of providers
 if (process.env.OIDC_NAME) {
+  let userinfo = {};
+  if (process.env.OIDC_FORCE_USER_INFO) {
+    userinfo = {
+      ...userinfo,
+      async request(context: any) {
+        return await context.client.userinfo(context.tokens.access_token ?? "");
+      },
+    };
+  }
+  if (process.env.OIDC_USER_INFO_URL) {
+    userinfo = {
+      ...userinfo,
+      url: process.env.OIDC_USER_INFO_URL,
+    };
+  }
   providers.push({
     id: process.env.OIDC_NAME,
     name: process.env.OIDC_NAME,
     type: "oauth",
     wellKnown: process.env.OIDC_DISCOVERY_URL,
-    authorization: { params: { scope: "openid" } },
+    authorization: { params: { scope: process.env.OIDC_SCOPES ?? "openid" } },
```

Comment:
The scope should be set by the app. We need `openid email profile` as otherwise this would not work.

Reply:
This is a good idea, even if some provider may not use `email profile` most should. This is a good default value, thanks.